bridge-fungible-token/src/utils.sw

```diff
@@ -69,7 +69,7 @@ fn shift_decimals_right(bn: U256, d: u8) -> Result<U256, BridgeFungibleTokenErro
     // shift decimals in increments of the max power of 10 that bn_div will allow (10^9)
     while (decimals_to_shift > 9u32) {
         // note: 1_000_000_000 = 10.pow(9)
-        let (adjusted, remainder) = bn_div(bn_clone, 1_000_000_000u32);
```

Comment:
Wouldn't this break things? `bn_clone` was being continuously divided, but now the code just divides the same original number over and over (see line 77)

Reply:
Ignore this change, locally I haven't actually removed this line, so there's some git weirdness going on in this PR's current state. The debug logs like `log(111)` don't exist locally anymore, but they're showing up here for some reason.